docs/docs-content/clusters/edge/edgeforge-workflow/appliance-studio.md

```diff
@@ -0,0 +1,203 @@
+---
+sidebar_label: "Appliance Studio"
+title: "Appliance Studio"
+description: "Overview of Appliance Studio."
+hide_table_of_contents: false
+sidebar_position: 1
+tags: ["edge"]
+---
+
+Appliance Studio is a lightweight and standalone configuration Graphic User Interface (GUI) for EdgeForge. It runs
+locally on your machine and allows you to build, save, edit, and manage the two configuration files that are essential
```

Comment:
Is it worth pointing out that this can be run on Linux, Windows, or Mac?

Just thinking that it might be good to highlight this somewhere.

Reply:
The application is containerized and the deployment methods we mention are both in virtualized environment,s so I feel like the OS is not particularly relevant? Like it technically is a linux container but you can run it on windows because Docker virtualized the environment.

Reply:
Agreed that we don't need to add it as it's a container.

I just thought it may look good to customers that "hey, we created this thing that you can run on any OS". It's not essential though.